app/helpers/textual_mixins/textual_devices.rb

```diff
@@ -42,11 +42,11 @@ def disks_attributes
       location = disk.location
       size = disk.size
       prov = disk.used_percent_of_provisioned
-      device_name = _("Hard Disk (%{controller_type} %{location}), Size %{size}, Percent Used Provisioned Space %{space}") %
-                       {:controller_type => ctrl_type,
-                        :location        => location,
-                        :size            => size,
-                        :space           => prov}
+      device_name = _("Hard Disk (%{controller_type} %{location}), Size %{size},
+                      Percent Used Provisioned Space %{space}") % {:controller_type => ctrl_type,
```

Comment:
@jzigmund, I am very unsure about what happens in the gettext catalog if you split a string across multiple lines. Are you sure whis will result in the same key as before? Or shall this be rather resolved with a backslash?